pkg/controller/apiextensions/composite/composed/composed.go

```diff
@@ -150,14 +150,11 @@ func (r *Composer) Compose(ctx context.Context, cp resource.Composite, cd resour
 		}
 	}
```

Comment:
Is there a benefit to getting the current resource while we're continuously configuring it?

Reply:
Not anymore since now we always unmarshal from the template.